e2e-tests/desktop/tests/scope.spec.js

```diff
@@ -54,7 +54,7 @@ test.describe('Scope tests', () => {
     await expect(headerNavLocator).toBeVisible();
     await expect(
       headerNavLocator.locator('.hds-dropdown-toggle-button'),
-    ).toHaveText('Global');
+    ).toHaveText('Global', { ignoreCase: true });
```

Comment:
just curious- was this addition meant to address a failing test? I noticed we didn't change any labels, so I was wondering why we had add the ignoreCase for this text.